.changeset/bright-rocks-stare.md

```diff
@@ -0,0 +1,5 @@
+---
+'@skeletonlabs/skeleton-svelte': patch
+---
+
+Make Svelte Slider marker snippet take the marker value as argument to enable marker customization
```

Comment:
@Akkretion looks like you accidently generated two Changesets. Let's nuke this one as the other includes the prefixed description per our requirements. Once that's done this is ready to merge. Thanks!